README.md

```diff
@@ -13,79 +13,228 @@
-    fn detect(&self, context: DetectContext<Self>) -> libcnb::Result<DetectResult, Self::Error> {
+    fn detect(&self, _context: DetectContext<Self>) -> libcnb::Result<DetectResult, Self::Error> {
         Ok(DetectResultBuilder::pass().build())
     }
 
-    // Similar to detect, this method will be called when the CNB lifecycle executes the 
+    // Similar to detect, this method will be called when the CNB lifecycle executes the
     // build phase.
     fn build(&self, context: BuildContext<Self>) -> libcnb::Result<BuildResult, Self::Error> {
         println!("Hello World!");
         println!("Build runs on stack {}!", context.stack_id);
-        Ok(BuildResultBuilder::new().build())
+
+        Ok(BuildResultBuilder::new()
+            .launch(Launch::new().process(Process::new(
+                process_type!("web"),
+                "echo",
+                vec!["Hello World!"],
+                false,
+                true,
+            )))
+            .build())
     }
 }
 
 // Implements the main function and wires up the framework for the given buildpack.
 buildpack_main!(HelloWorldBuildpack);
+
+```
+
+### Packaging the Buildpack
+Now that our buildpack is written, it's time to package it, so that it can be run. If you followed the steps to setup
+your development environment, you have access to the `libcnb` Cargo command that will handle packaging for you.
+
+In your project directory, run `cargo libcnb package` to start packaging:
+
+```shell
+$ cargo libcnb package
+INFO - Reading buildpack metadata...
+INFO - Found valid buildpack with id "libcnb-examples/my-buildpack" @ 0.1.0!
+INFO - Building buildpack binary (x86_64-unknown-linux-musl)...
+   Compiling syn v1.0.82
+   Compiling bit-vec v0.6.3
+   Compiling regex-syntax v0.6.25
+   # Omitting further compilation output...
+    Finished dev [unoptimized + debuginfo] target(s) in 19.71s
+INFO - Writing buildpack tarball...
+INFO - Successfully wrote buildpack tarball target/libcnb-examples_my-buildpack_0.1.0_dev.tar.gz (6.5M)
+INFO - Packaging successfully finished!
+INFO - Hint: To test your buildpack locally with pack, run: pack build my-image -b target/libcnb-examples_my-buildpack_0.1.0_dev.tar.gz --path /path/to/application
 ```
+
+If you get errors with hints about how to install required tools to cross-compile from your host platform to the
+target platform, follow them and try again. These differ from platform to platform, that's why they're not part of this
+quick start guide.
+
+### Running the Buildpack
+You might have seen in the hint in the output of the packaging step that we're now ready to run our buildpack locally 
+with `pack`. Before we can do this, we need an application to build. Since our buildpack does not interact with the
+application code at all, we just create an empty directory and use that as our application:
+
+```shell
+$ mkdir bogus-app
+$ pack build my-image -b target/libcnb-examples_my-buildpack_0.1.0_dev.tar.gz --path bogus-app
```

Comment:
Will this fail if the user's pack config still has the initial default builder? (Which I think might be the samples builder, I can't remember)

If so, we either need to pass `--builder heroku/buildpacks:20` here, or have making it the default (https://buildpacks.io/docs/tools/pack/cli/pack_config_default-builder/) be part of the initial setup section earlier.

Reply:
Good catch, I didn't try. I'll figure this out and update the docs accordingly.

Reply:
I added an explicit builder flag in the guide. Again, great catch!